sdk/resourcemanager/Azure.ResourceManager/src/ArmOperationOfT.cs

```diff
@@ -39,6 +39,7 @@ internal ArmOperation(IOperationSource<T> source, ClientDiagnostics clientDiagno
         /// <summary> Initializes a new instance of ArmOperation. </summary>
         public ArmOperation(ArmClient client, string id)
         {
+#if NET7_0_OR_GREATER
```

Comment:
Why is this NET7+? This code does not seem to be using the interface with static members.

Reply:
The method invoked through reflection is using the static method from the interface.

Reply:
If we are using reflection to invoke it, we don't need the interface, do we? i.e. if we are using reflection, we can just call a static method with the name/signature of our choosing. We don't need to bother with interfaces with static methods, and then this whole feature would work in NetStandard2.0. The whole point of adding this interfaces was to avoid reflection.